src/DynamoRevit/DynamoRevit.cs

```diff
@@ -305,15 +305,44 @@ public Result ExecuteCommand(DynamoRevitCommandData commandData)
             //subscribe to the assembly load
             AppDomain.CurrentDomain.AssemblyLoad += AssemblyLoad;
 
+            // Launch main Dynamo directly when ShowUiKey is true.
```

Comment:
`when ShowUiKey is false`?

Reply:
when ShowUiKey is false, it will go to else block in line 392(new file). We are changing the default parameter in CheckJournalForKey() to check if that flag is true or false or not present.